test/index.js

```diff
@@ -0,0 +1,38 @@
+var lastChar = require('../lib')
+  , assert = require('assert')
+  ;
+
+describe('last-char', function () {
```

Comment:
Great! Thanks. 

I opened this issue: https://github.com/IonicaBizau/code-style/issues/4
We can continue the conversation there. :smile: